Documentation/quickstart.md

```diff
@@ -0,0 +1,37 @@
+# Quickstart - Hubble in 3 Minutes or Less
+
+## 0. Requirements
+
+ * [minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/)
+ * [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+ * [Helm v3](https://github.com/helm/helm/releases/tag/v3.2.0)
+
+## 1. Start minikube
+
+    % minikube start --network-plugin=cni
+
+## 2. Deploy Cilium
+
+Deploy Cilium with Hubble enabled:
+
+    wget https://github.com/cilium/cilium/archive/master.zip
```

Comment:
i'll update this part to use https://helm.cilium.io/ once the next version of cilium gets released.